### PR TITLE
Feature/device push identifier

### PR DESCRIPTION
**Pull request by jkoenig134**
This is a preparation for a breaking change that will come to backbone v5.

### CODE REVIEW

File: packages/app-runtime/src/multiAccount/MultiAccountController.ts [hunk 1]
```diff
@@ -253,4 +253,27 @@ export class MultiAccountController {
...
+        const localAccount = LocalAccount.from(document);
+        localAccount.devicePushIdentifier = devicePushIdentifier;
+
+        await this._localAccounts.update(document, localAccount);
+    }
+
+    public async getAccountReferenceForDevicePushIdentifier(devicePushIdentifier: string): Promise<string> {
+        const document = await this._localAccounts.findOne({ devicePushIdentifier });
+        if (!document) {
+            // TODO: better error to make sure that the identifier could not be resolved
```
Comment (Milena-Czierlinski) [inline packages/app-runtime/src/multiAccount/MultiAccountController.ts:272]: TODO comment

Comment (jkoenig134) [reply packages/app-runtime/src/multiAccount/MultiAccountController.ts:272]: good catch!!

**Review by Milena-Czierlinski — COMMENTED** [review]
Missing version bumps

**Comment by jkoenig134** [comment]
Nope, I don't plan to deploy that.

**Review by Milena-Czierlinski — APPROVED** [review]

**Pull request merged**